Makefile

```diff
@@ -682,6 +689,7 @@ endif
 
 test: $(TARGETS) $(EXTRA_TARGETS)
 	+cd tests/simple && bash run-test.sh $(SEEDOPT)
+	+cd tests/simple_abc9 && bash run-test.sh $(SEEDOPT)
```

Comment:
This looks like an unrelated change?

Reply:
uf, my bad, let me check, guess I did not update something well